docs/DeveloperGuide.adoc

```diff
@@ -366,52 +366,50 @@ _Figure 4.1.2g: `UndoRedoStack` activity diagram_
 
 These two commands require one compulsory argument which is the amount that is borrowed/paid back. Indicating the index (as listed in the person list panel on the left side of the application window) of the person who borrowed or paid back money is optional. If no index is specified, the command will be executed for the person that is currently selected in the person list.
-The arguments (index and amount borrowed/paid back) are separated by a whitespace instead of special prefixes (e.g. prefix `p/` used for phone number). Hence, the `String#split` method is used to tokenize the input using a single whitespace as the delimiter. As seen in Figure 4.2.1, the tokenized inputs (index and amount borrowed) are then converted to their appropriate `Object` types and supplied as arguments to the `BorrowCommand` constructor. Input for the `payback` command is tokenized in the same manner.
+The arguments (index and amount borrowed/paid back) are separated by a whitespace instead of special prefixes (e.g. prefix `p/` used for phone number). Hence, the `String#split` method is used to tokenize the input using a single whitespace as the delimiter. As seen in Figure 4.2.1, the tokenized inputs (index and amount borrowed) are then converted to their appropriate `Object` types and supplied as arguments to the `BorrowCommand` constructor. Input for the `payback` command is tokenized and supplied to the `PaybackCommand` constructor in the same manner.
 
 The `BorrowCommand` and `PaybackCommand` are executed in `LogicManager`.
-`BorrowCommand` updates the `debt` and `totalDebt` attributes in the target `Person` object in the `AddressBook` class (which is in the `Model` component). A new `Person` object is created with the updated `debt` and `totalDebt` amount. The target `Person` object is then replaced with this new `Person` object.
-`PaybackCommand` updates the target person's debt in the same manner as the `BorrowCommand`. However, only the `debt` attribute is updated.
+`BorrowCommand` updates the `debt`, `totalDebt` and `dateRepaid` attributes in the target `Person` object through the `Model#addDebtToPerson()` which calls `AddressBook#addDebtToPerson()`. A new `Person` object is then created with the updated `debt` and `totalDebt` amount. Also, the `dateRepaid` field is set to `NOT REPAID`.  The target `Person` object is then replaced with this new `Person` object.
+
+`PaybackCommand` updates the target person's debt in the same manner as the `BorrowCommand`. However, only the `debt` attribute is updated. If the person has fully repaid his/her debts, the `PaybackCommand` will set the date repaid to the day the `PaybackCommand` was executed. The person will also be listed in the whitelist.
```

Comment:
i think you didn't see my prev comment? the person will not be listed in the whitelist if this is done on a blacklisted person